QueenCam/QueenCam/Presentation/Views/Reference/CloseView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CloseView.swift
+//  QueenCam
+//
+//  Created by 윤보라 on 10/14/25.
+//  Camera Preview에서 레퍼런스를 Fold 했을때 나타나는 View
+
+import SwiftUI
+
+struct CloseView: View {
```

Comment:
```suggestion

import SwiftUI

///  Camera Preview에서 레퍼런스를 Fold 했을때 나타나는 View
struct CloseView: View {
```

이렇게 주석달고 싶은 객체 바로 위에 `///`이나 `/** */`로 주석을 달면 Swift Documentation을 통해서 자동으로 문서화됩니다. 마치 SwiftUI 객체에 마우스 호버링하면 설명나오는 것처럼요!!

https://nshipster.com/swift-documentation/

근데 이건 그냥 제안이고 바꿀 곳이 너무 많다 싶으면 바꾸지 않으셔도 될 것 같아요

Reply:
이후 코드들에 반영하도록 하겠습니다. 감사합니다:)